ask-sdk-model-runtime/README.rst

```diff
@@ -0,0 +1,52 @@
+====================================================
+ASK SDK Model Runtime - Base components of SMAPI SDK
+====================================================
+
+ask-sdk-model-runtime is the package for Alexa Skills Kit (ASK) by
+the Software Development Kit (SDK) team for Python. It provides the
+base components and default implementations which work as the boiler
+plate code for developing Alexa Skills.
```

Comment:
We can provide a little bit more information on what this package is about. For ex, something like 

````
ask-sdk-model-runtime is the common models package that provides common implementations like Serializer, ApiClient etc. that is used by the SDK models.
````

Reply:
Ok